osbenchmark/worker_coordinator/runner.py

```diff
@@ -1254,6 +1254,17 @@ def _get_field_value(content, field_name):
                     return _get_field_value(content["_source"], field_name)
                 return None
 
+            def binary_search_for_last_negative_1(neighbors):
```

Comment:
I'd rather use a Python utility here but I couldn't find one in the standard library. Any suggestions?

Reply:
Im not aware of anything. Also, given that the list may not be sorted, it might be harder to do. That being said, I think this is good.